script.js

```diff
@@ -0,0 +1,64 @@
+$(document).ready(function(){
+    const movieGluAPI = '5zwPt8bzCu7r7rifnKoCd827XemsTLcO6rk51vTy';
+    // const NYTMovie = '';//we will need to grab the movie name from wherever it will be rendered on the page
+    // const NTYQueryMovies = 'https://api.nytimes.com/svc/movies/v2/reviews/search.json?query='+ NYTMovie + 'api-key=' + NYTAPI;
+    const TMDBAPI = '0adfd846cf8f1168484b5da4e5339d7c';
+    const personSearch = 'matthew mcconaughey' ;//function around get requests that gets value of input when search is clicked
+    const TMDBPersonID ='https://api.themoviedb.org/3/search/person?api_key=' + TMDBAPI + '&language=en-US&query=' + personSearch;
+    
+
+    // NYT get request
+    // $.get(NTYQueryMovies, function(response){
+        
+    
+    // })
+    //grab whatever search button is going to be called, run get request on click rather than on load
+    $().on('click', function(){
+        //input value needs to be what the actual input will be called
+        //personSearch = input.val();
+        retreiveData();
+    })
+
+    function retreiveData (){ //this retreives the person ID so we can use other API searches to get more movies
+        $.get(TMDBPersonID, function(response){
```

Comment:
```suggestion
        $.get(TMDBPersonID).then(function(response) {code here})
```
It's usually safer to call the function after we know we got an API response. This might end up running the function with no response if the endpoint is slow.

Reply:
I thought when using $.get it automatically did the function after the get request, almost as if the .then was built in? I could be wrong